src/client/src/utils/spaceKeyHandler.ts

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import {KEY_EVENTS} from "./constants"
+
+// document.documentElement.addEventListener('keydown', (e) => {
+//   if (e.key === KEY_EVENTS.SPACE) e.preventDefault();
+// })
+
+const spaceKeyHandler = (e: React.KeyboardEvent<HTMLAnchorElement>) => {
+
+  let anchor = e.target as HTMLAnchorElement;
+
+  //if pressing space on an anchor, prevent page scroll.
+  anchor.onkeydown = (e) => { 
```

Comment:
Curious, does e.stopPropogation() not work?   Wondering why you're attaching an event listener inside the event listener. Feels a bit odd. Can you explain why?

Reply:
e.stopPropogation pretty much just "pops" the event bubble, but all event handlers left on the current element (the anchor in this case) will still occur, so that doesn't work, but to be safe I just tested it and it doesn't work. 

The reason I did the inner event listener is because that way the event listener would only be added to elements that we select (whichever elements we add this handler to). The more generic option would be to add an event listener to all a tags with something like `document.getElementsByTageName("a");` I just thought this would be more localized and cause less potential side effects.